addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/handler/BigAssFanHandler.java

```diff
@@ -429,7 +502,7 @@ private void updateChannel(String channelName, State state) {
     }
 
     private String thingID() {
-        return thing.getUID().getId();
+        return thing.getUID().toString();
```

Comment:
The most places I quickly saw would also be fine if you would just pass `getUID()`, it will be auto converted for both loggers and string concatenation.